src/lm_buddy/jobs/evaluation/prometheus.py

```diff
@@ -107,25 +108,23 @@ def run_eval(config: PrometheusJobConfig) -> Path:
     # Instantiate OpenAI client to speak with the vLLM endpoint
     client = OpenAI(base_url=config.prometheus.inference.base_url)
 
-    # load dataset from W&B artifact
+    # Load dataset from W&B artifact
     hf_loader = HuggingFaceAssetLoader()
     dataset = hf_loader.load_dataset(config.dataset)
     if config.dataset.prompt_template is not None:
         dataset = format_dataset_with_prompt(
             dataset, config.dataset.prompt_template, config.dataset.text_field
         )
 
-    # get the tokenizer
+    # Get the tokenizer
     tokenizer_config = AutoTokenizerConfig(path=config.prometheus.inference.engine)
     tokenizer = hf_loader.load_pretrained_tokenizer(tokenizer_config)
 
-    # enable / disable tqdm
+    # Enable / disable tqdm
     dataset_iterable = tqdm(dataset) if config.evaluation.enable_tqdm else dataset
 
-    # open the output file for writing and iterate on samples
-    tracking_name = config.tracking.name if config.tracking is not None else "output.json"
```

Comment:
The `config.tracking.name` was a bug left in there by my previous refactors, that field no longer exists. Instead using `config.name` below to generate the directory path.